resources/assets/app.scss

```diff
@@ -2,3 +2,26 @@ $forge-path: "~@dosomething/forge/assets/";
 
 // Import Forge.
 @import "~@dosomething/forge/scss/forge";
+
+.tag {
+  list-style-type: none;
+  clear: both;
+  background: $light-gray;
+  color: $med-gray;
+  font-size: $font-regular;
+  font-weight: $weight-normal;
+  font-family: $font-proxima-nova;
+  text-align: center;
+  border: 0;
+  margin: 10px 10px 0 0;
+
+  &:hover {
+    background: $blue;
+    color: white;
+  }
+
+  &:active {
+    background: $blue;
+    color: white;
+  }
```

Comment:
It looks like it's displaying as a text input since there isn't a `type` attribute. Try adding a `type="button"` or switching the element type to `<button>...</button>`. Not sure if you want to handle it in this PR or not, but we'll definitely want to track whether a tag is active or not within React & update the classname accordingly.